examples/3d-heatmap/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const {resolve} = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+  entry: {
+    app: resolve('./app.js')
+  },
+
+  devtool: 'source-map',
+
+  module: {
+    rules: [{
+      // Compile ES2015 using buble
+      test: /\.js$/,
+      loader: 'buble-loader',
+      include: [resolve('.')],
+      exclude: [/node_modules/],
+      options: {
+        objectAssign: 'Object.assign'
+      }
+    }]
+  },
+
+  resolve: {
+    alias: {
+      // From mapbox-gl-js README. Required for non-browserify bundlers (e.g. webpack):
+      'deck.gl': resolve('./node_modules/deck.gl'),
```

Comment:
This needs to be consistent among all examples. Will make another request to address